modin/config/pubsub.py

```diff
@@ -75,6 +75,22 @@ class ExactStr(str):
         or (isinstance(value, str) and value.strip().isdigit()),
         help="an integer value",
     ),
+    dict: TypeDescriptor(
+        decode=lambda value: {
+            key.strip(): int(val) if val.strip().isdigit() else val.strip()
+            for key_value in value.split(",")
+            for key, val in [key_value.split("=")]
```

Comment:
```suggestion
            for key, val in [key_value.split("=", 1)]
```
otherwise the code would blow when `value` is something like `a=1=2,b=2`